spec/control_classes/tcp_client.rb

```diff
@@ -1,5 +1,61 @@
+require 'socket'
+
 class TCPClient
   include Client
-  def initialize(host, port); end
-  def send(file); end
-end
+  @socket
+  @out_dir
+
+  def initialize(host, port)
+
+    @out_dir = './files_to_transfer/'
+
+    @cannot_start = false
+
+    begin
+      @socket = TCPSocket.new(host, port)
+    rescue
```

Comment:
don't need error checking, since this is just a control and we just use it for benchmarking, we should just let it crash if something goes wrong